modules/ROOT/pages/index.adoc

```diff
@@ -2,16 +2,18 @@
 
 [IMPORTANT]
 ====
-OpenShift Serverless - Developer Preview releases contain features and functionalities that might not be fully tested. Customers are encouraged to provide feedback on Developer Preview releases.
+{serverlessproductname} - Technology and Developer Preview releases contain features and functionalities that might not be fully tested. Customers are encouraged to provide feedback on Technology and Developer Preview releases.
 
-Developer Preview releases are not production-ready, and customers are recommended to avoid using the project for production or business-critical workloads.
+Technology and Developer Preview releases are not production-ready, and customers are recommended to avoid using the project for production or business-critical workloads.
 
-If you need assistance with OpenShift Serverless - Developer Preview releases, please reach out on the following mailing list, where the Red Hat Development Team will be happy to assist you based on the availability and work schedules: serverless-interest@redhat.com
-====
+{serverlessproductname} provides information about additional, advanced use cases and integrations that are possible using OpenShift Container Platform and OpenShift Serverless, but are for technology and development purposes and are not officially supported by Red Hat.

```

Comment:
I think OpenShift Container Platform should be replaced with some variable `{ ... }` and OpenShift Serverless should be replaced with `{serverlessproductname}`.

The rest of this PR looks good. Thanks

Reply:
Thank you so much! I have added the attribute for the OpenShift Container Platform.

Reply:
@mgencur Can you please merge this PR now if everything is fine?